modules/force_rtl.py

```diff
@@ -30,7 +29,7 @@ def force_rtl(drone: dronekit.Vehicle) -> bool:
     rtl_command = generate_command.return_to_launch()
 
     # Change drone mode
-    drone.mode = dronekit.VehicleMode("RTL")
+    drone.mode = drone.set_flight_mode("RTL")
 
     # Utilize upload_command function to give RTL command to drone
     result = upload_commands.upload_commands(drone, [rtl_command], DRONE_TIMEOUT)
```

Comment:
this line is redundant. it drone.set_flight_mode("RTL") performs the same function as uploading an RTL command